modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIApplicationRelevantObjectEntryModelListener.java

```diff
@@ -47,6 +51,31 @@ public String getObjectDefinitionExternalReferenceCode() {
 		return "MSOD_API_APPLICATION";
 	}
 
+	@Override
+	public void onAfterCreate(ObjectEntry objectEntry)
+		throws ModelListenerException {
+
+		try {
+			_manageAPIApplication(objectEntry);
```

Comment:
I think this method is being used only once

Reply:
I will inline both of them, thanks @ccorreagg